pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -353,6 +353,15 @@
     "spdx": "MPL-2.0",
     "vendorHash": "sha256-XxltOTtCgmJ9wZX8Yw39HkwVVZb58kZjAH7jfKPhjKM="
   },
+  "doppler": {
+    "hash": "sha256-GhJ8wOsl696nBZwoTx7o1JzLP35CiTeUKP/d6LrzOqE",
+    "homepage": "https://registry.terraform.io/providers/DopplerHQ/doppler",
+    "owner": "DopplerHQ",
+    "repo": "terraform-provider-doppler",
+    "rev": "v1.2.3",
+    "spdx": "MPL-2.0",
```

Comment:
https://github.com/DopplerHQ/terraform-provider-doppler 

Where did you get the license from, upstream doesn't seem to have one in their repo? 

Almost all of the providers set their license automatically when they are updated based on what is used in the upstream repo.

Reply:
That's a good question. Tbh I just copied a block next to it and then changed the hash and assumed it too would be MPL2. I created DopplerHQ/terraform-provider-doppler#54 and hopefully some light can be shed on this.

Reply:
Ah, I didn't see your comment and have updated my PR to start from v1.2.4 to avoid any ambiguity.

Reply:
Thanks for getting this fixed upstream.